src/EditorFeatures/TestUtilities/PullMemberUp/TestPullMemberUpService.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using Microsoft.CodeAnalysis.CodeRefactorings.PullMemberUp;
+using Microsoft.CodeAnalysis.CodeRefactorings.PullMemberUp.Dialog;
+using Microsoft.CodeAnalysis.PullMemberUp;
+
+namespace Microsoft.CodeAnalysis.Test.Utilities.PullMemberUp
+{
+    internal class TestPullMemberUpService : IPullMemberUpOptionsService
+    {
+        private IEnumerable<(string member, bool makeAbstract)> SelectedMembers { get; }
```

Comment:
private auto-prop is strange and uncommon.  why not just a readonly field for both of these?